blaze-client/src/main/scala/org/http4s/blaze/client/BlazeClientBuilder.scala

```diff
@@ -74,7 +74,7 @@ sealed abstract class BlazeClientBuilder[F[_]] private (
     val chunkBufferMaxSize: Int,
     val parserMode: ParserMode,
     val bufferSize: Int,
-    val executionContext: ExecutionContext,
+    val executionContextConfig: ExecutionContextConfig,
```

Comment:
The param in the scaladoc no longer lines up.

I'm surprised this passed MiMa without a synthetic `def executionContext`.

Reply:
Lost my other comment: since this type is `private[http4s]`, I don't think it makes sense to expose it as a val.  Just a parameter.

Reply:
> I'm surprised this passed MiMa without a synthetic `def executionContext`.

Weird, that doesn't seem right 🤔

Reply:
Found the problem: https://github.com/http4s/http4s/pull/5215